spokestack/profile/vad_trigger_asr.py

```diff
@@ -0,0 +1,53 @@
+"""
+Pipeline profile with vad trigger and asr
+"""
+from spokestack.activation_timeout import ActivationTimeout
+from spokestack.asr.speech_recognizer import CloudSpeechRecognizer
+from spokestack.io.pyaudio import PyAudioInput
+from spokestack.pipeline import SpeechPipeline
+from spokestack.vad.webrtc import VoiceActivityDetector, VoiceActivityTrigger
+
+
+class VoiceActivityTriggerSpokestackASR:
+    """ VAD Trigger ASR """
+
+    def create(
```

Comment:
nice, since this class doesn't have any state, you could mark this method with `@staticmethod` and just call it like `VoiceActivityTriggerSpokestackASR.create()` without creating an instance

Reply:
TIL. thanks!!